build/images/README.md

```diff
@@ -0,0 +1,148 @@
+# Building Base Images
+
+This directory contains tooling for building base images for use as nodes in Kubernetes Clusters. [Packer](https://www.packer.io) is used for building these images. This tooling has been forked and extended from the [Wardroom](https://github.com/heptiolabs/wardroom) project.
+
+## Prerequisites
+
+### Hypervisor
+
+The images may be built using one of the following hypervisors:
+
+| OS | Builder |
+|----|---------|
+| Linux | VMware Workstation |
+| macOS | VMware Fusion |
```

Comment:
This should work with Windows as well right?

```suggestion
| macOS | VMware Fusion |
| Windows | VMware Workstation |
```

Reply:
Conceivably, but none of the build tooling or paths would work with Windows today.

Reply:
Fair enough, resolving